runtime/src/serde_snapshot.rs

```diff
@@ -314,7 +314,7 @@ where
                 std::fs::rename(append_vec_abs_path.clone(), target).or_else(|_| {
                     let mut copy_options = CopyOptions::new();
                     copy_options.overwrite = true;
-                    fs_extra::move_items(&vec![&append_vec_abs_path], &local_dir, &copy_options)
+                    fs_extra::move_items(&[&append_vec_abs_path], &local_dir, &copy_options)
```

Comment:
Do this in a different PR

Reply:
Ah, this looks familiar. Clippy was also suggesting this change on @t-nelson 's PR with the Cargo.lock craziness. I think it you follow his advice to reset that file, you'll be able to remove this change too.

Reply:
Could be solved by reverting the dependencies, as you said.